GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -80,12 +104,38 @@ extension CallVisualizer {
                 .present(viewController, animated: true)
         }
 
+        func showVideoCallViewController() {
+            createOperatorImageBubbleView()
+            bubbleView.alpha = 0.0
```

Comment:
Is setting alpha to zero, enough? Wouldn't it still be interactive, it terms of intercepting touches, thus preventing other views under it, to receive these touches?

Reply:
It turns out it wasn't needed anymore at all. I had made some changes prior and this was just redundant. fixed [54b8b08](https://github.com/salemove/ios-sdk-widgets/pull/494/commits/54b8b08379b18922451d03af2c4e497bac940b1a)